resources/qml/PrinterSelector/MachineSelector.qml

```diff
@@ -139,12 +146,13 @@ Cura.ExpandablePopup
         {
             id: connectionStatusTooltipHoverArea
             anchors.fill: parent
-            hoverEnabled: connectionStatusMessage !== ""
+            hoverEnabled: getConnectionStatusMessage() !== ""
```

Comment:
Why did you change this into a function? It should work when it's a property, right?

Reply:
Nope, it doesn't get re-evaluated when hovering over it. I had to change into a function, or else it kept the old message in the tooltip, until the machine was changed.